### DISCUSSION TITLE
Secure Connection Failed

### DISCUSSION THREAD

**Discussion by mcandemir** [vercel/community#5114] [Help]
### Question

Hi, I think I'm having a SSL problem. I tried to detect the exact root of the problem but it was beyond my knowledge. Does anyone faced with this before? Any ideas?

```
An error occurred during a connection to portfolio-mcandemir.vercel.app. PR_CONNECT_RESET_ERROR

Error code: PR_CONNECT_RESET_ERROR

-     The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
-     Please contact the website owners to inform them of this problem.

```

PS: I tried to open the website with a different IP and different browsers, but it didn't solved the problem.

**Comment by mcandemir** [comment]
Ok I think the root of the problem might be this since I live in Turkey too: https://github.com/orgs/vercel/discussions/5102

**Comment by amyegan** [comment]
Thanks for reporting this! We pinned [that discussion](https://github.com/orgs/vercel/discussions/5102) as the main thread for the issue and we'll continue posting updates there 🙇
